packages/vaadin-grid/src/vaadin-grid.js

```diff
@@ -919,6 +921,10 @@ class GridElement extends ElementMixin(
         cell._instance.setProperties(model);
       }
     });
+
+    this._updateDetailsCellHeight(row);
+
+    requestAnimationFrame(() => this.notifyResize());
```

Comment:
Is it a necessary call?

Reply:
Note that `notifyResize` is to be removed and currently does nothing https://github.com/vaadin/web-components/blob/master/packages/vaadin-grid/src/vaadin-grid.js#L1002-L1010

Reply:
Thanks for the note! Then I can probably remove this line at all...